source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,89 @@
+---
+owner_slack: "#modernisation-platform"
+title: Set Up Macie
+last_reviewed_on: 2024-04-18
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+# Introduction
+
+It is relatively easy to set up Macie for users and it need to be done on an account by account basis. **NOTE** Macie ONLY looks at S3 buckets and checks their access information so it will not look at databases or EC2 settings, for example.
```

Comment:
What does "checks their access information" mean?  Does it not scan their contents for certain data?

Reply:
I'm not sure. I will double check and amend this as needed.

Reply:
This hasn't been resolved the same comment still stands

Reply:
I was sure it has been removed. It has now.